moped-editor/src/views/projects/newProjectView/NewProjectView.js

```diff
@@ -250,32 +164,13 @@ const NewProjectView = () => {
   return (
     <Page title="New project">
       <Container>
-        <Card className={classes.cardWrapper}>
-          <CardContent>
```

Comment:
Small change but I removed the card wrapper since we were only using it to have the nice [Paper](https://mui.com/material-ui/react-paper/?srsltid=AfmBOopFAZDngnS6BNE7EwbyrKmamJvdq9DjIcp3Il4r-uSbhev6cGFm) background in this layout. The built-in card styles made the alignment of children components more difficult since it had inconsistent padding-bottom.